File: uva-713.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    //freopen("input.txt","r",stdin);
    string s,s1,res;
    int n;
    cin >> n;
    while(n--)
    {
    cin >> s >> s1;
    reverse(s.begin(),s.end());
    reverse(s1.begin(),s1.end());

    long long n,n1;
    istringstream ss(s);
    ss>>n;
    istringstream iss(s1);
    iss>>n1;
    
    int result = n+n1;
    res = to_string(result);
    reverse(res.begin(),res.end());
    long long l;
    istringstream ssp(res);
    ssp>>l;
    cout << l << endl;
    }
    return 0;
}

File: uva/uva-713.py
import sys
def main():
   # sys.stdin = open('input.txt')
    n = int(input())
    while(n>0):
        x,y = input().split()
        z = x[::-1]
        m = y[::-1]
        r = (int(z)+int(m))
        res = str(r)
        l = (res[::-1])
        print(int(l))
        n = n-1
if __name__ == "__main__":
    main()
